src/types.rs

```diff
@@ -12,6 +12,7 @@ pub type AccountId = String;
 pub type RawAddress = [u8; 20];
 pub type RawU256 = [u8; 32]; // Little-endian large integer type.
 pub type RawH256 = [u8; 32]; // Unformatted binary data of fixed length.
+pub type Gas = u64;
```

Comment:
Just a note here, this should be applied to other precompiles and elsewhere. A Newtype likely may be preferred to ensure that there is no accidental mixing with other u64. Not recommended for now.

Reply:
Yeah, a good idea to have a distinct type for gas.

Reply:
> Not recommended for now.

👀 
It is very easy change, and it is ok to introduce it from the beginning, why not now?

Reply:
Just out of scope to add them to all precompiles. You can do it, but probably after this PR is merged.

Reply:
Exactly, let's keep PRs atomic. That will also help avoid merge conflicts given active work by others.